sympy/solvers/inequalities.py

```diff
@@ -393,6 +393,9 @@ def solve_univariate_inequality(expr, gen, relational=True, domain=S.Reals, cont
         A Relational type output is expected or not
     domain : Set
         The domain over which the equation is solved
+    cont_domain: bool
```

Comment:
I don't really have a strong opinion on the matter either. I suppose `continuous` does make sense and is probably easier to type.